cpp/include/core/MctsConfigurationBase.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+namespace core {
+
+struct  MctsConfigurationBase {
+  static constexpr float kOpeningLength = 0;
```

Comment:
I think it's better to delete this line. The concept definition dictates that `kOpeningLength` is needed, and 0 is not a good default.